### PR TITLE
Improve publication page

### PR DESCRIPTION
**Pull request by jfan0010**
# Description 
Minor modification to publication page and dashboard layout to fix the overlapping of page content and sidebar, as well as the display of twitter panel.

# Type of change 
- [x] Bug fix
- [ ] New feature
- [ ] Refactoring
- [ ] Documentation 

# Problem
See [#issues in Slack](https://fit3170fy2021-ib99389.slack.com/archives/C02ABMVGL2D/p1628972487001800)

# Solution description
As above. See commits for details.

Attach screenshot of the expected result (e.g. UI of the website/ postman api call result/ database changes)

image

# Risk
Mild. Only affect UI. Let me know if there's any page that got messed up because of this PR.

### CODE REVIEW

**Review by pmyip3 — APPROVED** [review]

**Review by vgoh1 — APPROVED** [review]
Finally the publications component won't clip into the sidebar when I have the console open. :) Thanks for fixing this!

File: client/src/components/layouts/layoutComponents/Sidebar.js [hunk 1]
```diff
@@ -3,17 +3,18 @@
...
 
   return (
-    <div className="Sidebar">
+    <Fragment>
       <ul className="SidebarList">
         {data.map((val, key) => {
           return (
             // if the link property of a sidebar item is undenfined, stays in the current page
-            <Link to={val.link ? val.link : '#'} key={key}> 
+            <Link to={val.link ? val.link : '#'} key={key}>
```
Comment (aksh0001) [inline client/src/components/layouts/layoutComponents/Sidebar.js:17]: Would it be possible to eliminate the return on L15? By just returning a block enclosed by parentheses? Should make it more readable

**Comment by aksh0001** [comment]
According to the PR description, the icons on a publication seem to have floated off: like the arrow and the three dots. Is this expected?

File: client/src/components/layouts/layoutComponents/Sidebar.js (hunk 1 above)
Comment (aksh0001) [reply client/src/components/layouts/layoutComponents/Sidebar.js:17]: Also could you please fix the typo for undefined In the L16 comment/improve the comment

**Review by aksh0001 — APPROVED** [review]
LGTM, non-blocker comments to be addressed.

File: client/src/components/layouts/layoutComponents/Sidebar.js (hunk 1 above)
Comment (jfan0010) [reply client/src/components/layouts/layoutComponents/Sidebar.js]: done

**Comment by jfan0010** [comment]
> According to the PR description, the icons on a publication seem to have floated off: like the arrow and the three dots. Is this expected?
Right, that is an existing issue when resizing the screen. Will look into it in the coming ticket, cause this PR is mainly the general layout and the twitter box.

**Pull request merged**